pkg/datapath/linux/config/config.go

```diff
@@ -105,6 +107,17 @@ func (h *HeaderfileWriter) WriteNodeConfig(w io.Writer, cfg *datapath.LocalNodeC
 			ipv4NP := node.GetNodePortIPv4()
 			cDefinesMap["IPV4_NODEPORT"] = fmt.Sprintf("%#x", byteorder.HostSliceToNetwork(ipv4NP, reflect.Uint32).(uint32))
 		}
+
+		pm := probes.NewProbeManager()
+		supportedMapTypes := pm.GetMapTypes()
+		if supportedMapTypes.HaveLruHashMapType {
+			cDefinesMap["IPV4_FRAGMENTS"] = "1"
+			cDefinesMap["IPV4_FRAG_DATAGRAMS_MAP"] = fragmap.MapName
+			cDefinesMap["CILIUM_IPV4_FRAG_MAP_MAX_ENTRIES"] = fmt.Sprintf("%d", fragmap.MaxEntries)
+			// } else {
+			// TODO: Log something? But "log" not imported in this file yet
+			// log.Warningf("No support for IPv4 fragments (LRU maps not supported by kernel)")
```

Comment:
This logging can be performed when we are initializing the daemon, around the same place where we perform the bpf probing. For example here: https://github.com/cilium/cilium/blob/e1c05e24814c5aff774305836b2c2e825793b272/pkg/datapath/linux/requirements.go#L175-L184